src/main.cpp

```diff
@@ -2063,13 +2064,14 @@ static int64_t nTimePostConnect = 0;
  * corresponding to pindexNew, to bypass loading it again from disk.
  */
 bool static ConnectTip(CValidationState &state, CBlockIndex *pindexNew, const CBlock *pblock) {
+    const CChainParams& chainparams = Params();
```

Comment:
Yep, but chainparams is used in more places than chainParams (my fault) so it seems too late. I explained this in more detail here: https://github.com/bitcoin/bitcoin/pull/6163#discussion_r38074977